web-comp.js

```diff
@@ -0,0 +1,204 @@
+const catalogProducts = [
+    {
+        id: 1,
+        imageURL: "images/product/ak101-img.jpeg",
+        altAlert: " it's an ak101",
+        name: "ak101",
+        description: "the ak101 it's a very curious rifle because it combines the look of the ak with the fire power of his historical enemy, chambered in 5.56x45 the ak101 provides one of the most armor piercing experience",
+        category: "",
+        price: 299.99
+    },
+    {
+        id: 2,
+        imageURL: "images/product/ak12-img.jpeg",
+        altAlert: " it's an ak12",
+        name: "ak12",
+        description: "chose in 2019 for becoming the issued rifle of the russian army the ak12 is a very reliable weapon, thankfully to his adjustable stock and the added rails that can allow the user to utilize ordinary aiming systems has become one of the greatest ak ever invented",
+        category: "",
```

Comment:
I planned to use the category for filters, if they arent implemented then remove them rather than defining them empty. (Avoid leaving unused code)

Reply:
fixed